wiki/en/Software-Manual.md

```diff
@@ -119,14 +117,70 @@ You can group users together using the "group" toggle. Moving the fader of any m
 
 Note that muting your **own** channel only means you will not hear your signal from the server (and is not advised as it can lead to you becoming out of time with other players). This is therefore not the same as using "[Mute Myself](#mute-myself-button)".
 
 ### Solo button
 
 Allows you to hear one or more users on their own. Those not soloed will be muted. Note also that those people who are not soloed will see a "muted" icon above your fader.
 
+## Menu commands
+
+### File > Connection Setup...
+
+Opens the [connection dialogue (see above)](#connectdisconnect-button).
+
+### File > Load/Save Mixer Channels Setup
+
+You can save and restore the mix you have for your band rehearsals (fader, mute, pan, solo etc.) and load these any time (even while you are playing). Loading can also be done by drag/drop to the mixer window.
+
+### File > Exit
+
+Closes all the windows and exits the application.
+
+### Edit > Clear All Stored Solo/Mute Settings
+
+Whenever you Solo or Mute a channel, Jamulus stores the settings against them, so that if they leave and rejoin - or you join a server where they are playing - the settings will automatically be applied.  If you want clear _all_ stored settings, you can do it here.
+
+### Edit > Set All Faders to New Client Level
+
+Applies the [configured "New Client Level"](#new-client-level) to all currently connected channels.
+
+### Edit > Auto-Adjust All Faders
+
+Applies a one-off fader setting to each channel depending on its volume. Useful for large ensembles to get a reasonable overall mix, although individual adjustments might still be necessary. Best applied during a warm-up or a uniform part of the music piece.
+
+### View > (sort options)
+
+Most items under the "View" menu allow you to adjust the order Jamulus chooses to display channels on the server:
+* Own Fader First<br/>
+This option can be used in addition to the others to move your own channel to always be the leftmost.
+
+* No user sorting<br/>
+This does not take any user details into account when sorting.  It sorts by the order channels join the current server, with new channels being added to the right-hand end.
+* Sort by Name<br/>
+Sorts by the name someone has chosen in their profile.
+* Sort by Instrument / City<br/>
+Sorts by the instrument or city someone has in their profile, along with their name.
+* Sort by Group<br/>
+Where the fader group feature is in use, this sorts in ascending group number from left to right (and within that, by name), with all ungrouped channel off to the right.
+* Sort by Server Channel<br/>
+Where Jamulus channel controls (fader, mute, solo, etc) are being controlled by MIDI (see [Using --ctrlmidich for MIDI controllers](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)), this sorts by the server-assigned channel number to help ensure a stable sort order that aligns with MIDI hardware controls.
```

Comment:
Beyond fader, pan, mute, and solo, what does etc. cover?

Reply:
Just "Mute myself", I think.